docs/meson.build

```diff
@@ -1,11 +1,11 @@
 tag_file_modules = [
   'mm-common-libstdc++', # provided by mm-common
-  'sigc++-2.0',
-  'glibmm-2.4',
+  'sigc++-3.0',
+  'glibmm-2.68',
   'cairomm-1.0',
   'pangomm-1.4',
   'atkmm-1.6',
-  'gtkmm-3.0',
+  'gtkmm-4.0',
 ]
 
```

Comment:
`meson` is showing me these warnings:

```
NOTICE: Future-deprecated features used:
 * 0.55.0: {'ExternalProgram.path'}
 * 0.56.0: {'dependency.get_pkgconfig_variable'}
```

Is this something that you would like to fix now? If not, I will take a look at it after merging this PR in

Reply:
That's fine, thanks